src/app/components/Counter/Counter.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import styled from 'styled-components'
+import ChevronDown from '../../Icons/ChevronDown'
+import ChevronUp from '../../Icons/ChevronUp'
+
+type CounterProps = {
+  value: number
+  onIncrement: () => void
+  onDecrement: () => void
+}
+
+export default function Counter({
+  value,
+  onIncrement,
+  onDecrement,
+}: CounterProps): JSX.Element {
+  return (
+    <CounterContainer>
+      <VoteButton displayType="up" onClick={onIncrement}>
+        <ChevronUp />
+      </VoteButton>
+      <ValueView>{value}</ValueView>
+      <VoteButton displayType="down" onClick={() => value > 0 && onDecrement()}>
```

Comment:
Why does the value have to be greater than zero? :D interesting, maybe we could talk about how this type of collective decision making works STS :)

Reply:
oh the counter should not go below zero thats all.